plugin/src/test/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConfigurationTest.java

```diff
@@ -8,6 +13,9 @@
     @Rule
     public JenkinsRule j = new JenkinsRule();
 
+    @Rule
+    public KubernetesServer k = new KubernetesServer(true, true);
+
```

Comment:
I'd create some more variations of the config w/ various parameters set to null or invalid. That'll give you the biggest bang for buck in terms of getting code and branch coverage up